src/CachePoolTest.php

```diff
@@ -153,19 +153,21 @@ public function testGetItems()
 
         $keys  = ['foo', 'bar', 'baz'];
         $items = $this->cache->getItems($keys);
-        $this->assertCount(3, $items);
+
+        $count = 0;
 
         /** @type CacheItemInterface $item */
         foreach ($items as $i => $item) {
             $item->set($i);
             $this->cache->save($item);
+
+            $count++;
```

Comment:
Incrementing the `$count` variable in this loop is pointless because you set it back to zero before ever reading from it.